macstadium-staging/main.tf

```diff
@@ -72,99 +31,43 @@ provider "vsphere" {
   allow_unverified_ssl = true
 }
 
-module "macstadium_infrastructure_staging" {
-  source                        = "../modules/macstadium_infrastructure_staging"
-  index                         = "${var.index}"
-  name_suffix                   = "${var.name_suffix}"
-  vanilla_image                 = "${var.macstadium_vanilla_image}"
-  datacenter                    = "pod-1"
-  cluster                       = "MacPro_Staging_1"
-  datastore                     = "DataCore1_1"
-  internal_network_label        = "Internal"
-  management_network_label      = "ESXi-MGMT"
-  jobs_network_label            = "${var.jobs_network_label}"
-  jobs_network_subnet           = "${var.jobs_network_subnet}"
-  jobs_network_mac_address      = "${var.jobs_network_mac_address}"
-  ssh_user                      = "${var.ssh_user}"
-  travisci_net_external_zone_id = "${var.travisci_net_external_zone_id}"
-  vsphere_ip                    = "${var.vsphere_ip}"
-  vm_ssh_key_path               = "${path.module}/config/travis-vm-ssh-key"
-}
-
-module "vsphere_janitor_staging_com" {
-  source                  = "../modules/vsphere_janitor"
-  host_id                 = "${module.macstadium_infrastructure_staging.wjb_uuid}"
-  ssh_host                = "${module.macstadium_infrastructure_staging.wjb_ip}"
-  ssh_user                = "${var.ssh_user}"
-  vsphere_janitor_version = "${var.vsphere_janitor_version}"
-  config_path             = "${path.module}/config/vsphere-janitor-staging-com"
-  env                     = "staging-com"
-  index                   = "${var.index}"
-}
-
-module "vsphere_monitor" {
-  source                  = "../modules/vsphere_monitor"
-  host_id                 = "${module.macstadium_infrastructure_staging.util_uuid}"
-  ssh_host                = "${module.macstadium_infrastructure_staging.util_ip}"
-  ssh_user                = "${var.ssh_user}"
-  vsphere_monitor_version = "${var.vsphere_monitor_version}"
-  config_path             = "${path.module}/config/vsphere-monitor"
-  index                   = "${var.index}"
-}
+module "kubernetes_cluster" {
+  source                 = "../modules/macstadium_k8s_cluster"
+  name_prefix            = "cluster-staging"
+  ip_base                = 100
```

Comment:
hey what is this ip_base value?  I looked in K8s terraform provider but no luck.

Reply:
It's the starting point for the IPs we assign to the cluster VMs. So the master will be .100 and the nodes will be .101 and .102 in this case.